provider/cmd/pulumi-resource-aws/schema.json

```diff
@@ -440295,7 +440295,7 @@
entityProvider(\"graph.facebook.com\")\n                .ambiguousRoleResolution(\"AuthenticatedRole\")\n                .type(\"Rules\")\n                .mappingRules(IdentityPoolRoleAttachmentRoleMappingMappingRuleArgs.builder()\n                    .claim(\"isAdmin\")\n                    .matchType(\"Equals\")\n                    .roleArn(authenticatedRole.arn())\n                    .value(\"paid\")\n                    .build())\n                .build())\n            .roles(Map.of(\"authenticated\", authenticatedRole.arn()))\n            .build());\n\n    }\n}\n```\n```yaml\nresources:\n  mainIdentityPool:\n    type: aws:cognito:IdentityPool\n    properties:\n      identityPoolName: identity pool\n      allowUnauthenticatedIdentities: false\n      supportedLoginProviders:\n        graph.facebook.com: '7346241598935555'\n  authenticatedRole:\n    type: aws:iam:Role\n    properties:\n      assumeRolePolicy: ${authenticatedPolicyDocument.json}\n  authenticatedRolePolicy:\n    type: aws:iam:RolePolicy\n    properties:\n      role: ${authenticatedRole.id}\n      policy: ${authenticatedRolePolicyPolicyDocument.json}\n  mainIdentityPoolRoleAttachment:\n    type: aws:cognito:IdentityPoolRoleAttachment\n    properties:\n      identityPoolId: ${mainIdentityPool.id}\n      roleMappings:\n        - identityProvider: graph.facebook.com\n          ambiguousRoleResolution: AuthenticatedRole\n          type: Rules\n          mappingRules:\n            - claim: isAdmin\n              matchType: Equals\n              roleArn: ${authenticatedRole.arn}\n              value: paid\n      roles:\n        authenticated: ${authenticatedRole.arn}\nvariables:\n  authenticatedPolicyDocument:\n    fn::invoke:\n      Function: aws:iam:getPolicyDocument\n      Arguments:\n        statements:\n          - effect: Allow\n            principals:\n              - type: Federated\n                identifiers:\n                  - cognito-identity.amazonaws.com\n            actions:\n              - sts:AssumeRoleWithWebIdentity\n            conditions:\n              - test: StringEquals\n                variable: cognito-identity.amazonaws.com:aud\n                values:\n                  - ${mainIdentityPool.id}\n              - test: ForAnyValue:StringLike\n                variable: cognito-identity.amazonaws.com:amr\n                values:\n                  - authenticated\n  authenticatedRolePolicyPolicyDocument:\n    fn::invoke:\n      Function: aws:iam:getPolicyDocument\n      Arguments:\n        effect: Allow\n        actions:\n          - mobileanalytics:PutEvents\n          - cognito-sync:*\n          - cognito-identity:*\n        resources:\n          - '*'\n```\n{{% /example %}}\n{{% /examples %}}\n\n## Import\n\nCognito Identity Pool Roles Attachment can be imported using the Identity Pool ID, e.g.,\n\n```sh\n $ pulumi import aws:cognito/identityPoolRoleAttachment:IdentityPoolRoleAttachment example us-west-2:b64805ad-cb56-40ba-9ffc-f5d8207e6d42\n```\n\n ",
```

Comment:
Can't figure out what's changed here ... or why it's changed - might just need to revert this.

Reply:
Oh, something related to the java version maybe?